### ISSUE TITLE
0.3.6 appears to be missing "where" helper function

### ISSUE THREAD

**Issue by ejhayes** [jwilder/docker-gen#58]
Looks like the [0.3.6](https://github.com/jwilder/docker-gen/releases/tag/0.3.6) release doesn't have `where` available. Any way to update 0.3.6 to include this?

```
unable to parse template: template: my_template:4: function "where" not defined
```

**Comment by md5** [comment]
Wouldn't it make sense to just have a 0.3.7 instead?

It does indeed look like a release is overdue: https://github.com/jwilder/docker-gen/compare/0.3.6...edcd195d283e16b93c65f7461a9c880b8080a658

It looks like #40, #39, #44, #45, #46, #49, #52, #55, and #56 have all been merged since the last release, in addition to the hand-merged changes for `where` from #54.

**Comment by jwilder** [comment]
I cut 0.3.7 which should resolves this now.
